package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-cimpress-comment",
-  "version": "0.6.5",
+  "version": "0.6.6",
```

Comment:
Can we also update CHANGELOG file?

Reply:
Done! The changelog is a good idea, I just need to get used to keeping it in mind.